### ISSUE TITLE
Object creation fails

### ISSUE THREAD

**Issue by CodeMonkeySteve** [briandoll/Rit#1]
When creating a Plate or Edition, I get:

```
NoMethodError (undefined method `first' for #<HashWithIndifferentAccess:0xb6e371d0>):
  lib/parses_plate_edition_parameters.rb:4:in `parse_params'
  ...
```

This is with Ruby 1.8.6.

**Comment by briandoll** [comment]
Kasima just pushed the fix for this: c60b9254c840c91ba3638e9d764d3b1b5986c584

**Comment by CodeMonkeySteve** [comment]
Confirmed, thanks.
